ykman/cli/fido.py

```diff
@@ -158,16 +159,20 @@ def reset(ctx):
     The reset must be triggered immediately after the YubiKey is
     inserted, and requires a touch on the YubiKey.
     """
-    controller = ctx.obj['controller']
+    click.echo('Remove and re-insert your YubiKey to perform the reset...')
+
+    removed = False
+    while True:
+        descriptors = list(get_descriptors())
+        n_keys = len(descriptors)
+        if not n_keys:
+            removed = True
+        if removed and n_keys == 1:
+            break
```

Comment:
How about a short sleep here (~50 ms or so), so we don't max the processor for this?